src/database/migrations/0040-create-changemaker_roles.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TYPE access_type_t AS ENUM (
+  'manage',
+  'edit',
+  'view'
+);
+
+CREATE TABLE changemaker_roles (
+  user_keycloak_user_id UUID NOT NULL,
```

Comment:
3. This order makes it feel more like the User is the subject and the Changemaker is the object (as an English speaker). If this is how it works, then I think it might be even clearer to flip the order of `changemaker_id` and `access_type` so that the table order is User, Access, Changemaker. And also make the TypeScript types match this same order to be consistent. Perhaps a comment on each of the User/Access/Object columns or a single comment on each Role table would help clarify the meaning. The same goes for the TypeScript types.

Reply:
That's a really good idea